minmodkg/api/models/public_mineral_site.py

```diff
@@ -19,6 +19,17 @@
 from minmodkg.services.kgrel_entity import EntityService
 from minmodkg.typing import IRI, InternalID
 from pydantic import BaseModel, Field
+from urllib.parse import urlparse
+
+
+def is_valid_url(url) -> bool:
+    if " " in url:
+        return False
+    try:
+        result = urlparse(url)
```

Comment:
urlparse won't throw exception if url is a string and we assume url is a string, so remove try & catch.